clinlp/entity.py

```diff
@@ -213,3 +214,41 @@ def __call__(self, doc: Doc):
         doc.set_ents(entities=ents)
 
         return doc
+
+
+def create_term(row, col_term):
+    """Creates a Term object with available attributes."""
+    term = Term("")
+    for attr, value in row.items():
+        if attr == col_term:
+            term.phrase = value
+        if value == value:
+            if value == "True":
+                value = True
+            elif value == "False":
+                value = False
+            elif isinstance(value, float):
+                value = int(value)
+            setattr(term, attr, value)
```

Comment:
Wat als er een kolom in de csv zit die geen attribute in Term is?

Reply:
Ik wilde de attributes niet hardcoden, omdat die misschien nog uitbreiden in de toekomst. Heb het nu veranderd zodat de attributes uit de Term class worden gehaald en in deze functie worden gebruikt in een if statement. Daarmee pakt hij alleen iets uit de csv als de kolomnaam overeenkomt met een Term attribute.